public/app/plugins/datasource/graphite/README.md

```diff
@@ -1,4 +1,4 @@
-# Graphite Datasource -  Native Plugin
+# Graphite Data Source -  Native Plugin
```

Comment:
For the READMEs, let's keep Title Case for now. This is fine.

Reply:
Thanks, noted.